### PR TITLE
Issue 249-Add skeleton FAT test for CDI2.0

### PR DESCRIPTION
**Pull request by neilgsyoung**

### CODE REVIEW

**Comment by neilgsyoung** [comment]
#build

File: dev/com.ibm.ws.cdi.2.0_fat/.classpath [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="fat/src"/>
+	<classpathentry kind="src" path="test-applications/beanManagerLookupApp/src"/>
+	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="lib" path="C:/Users/IBM_ADMIN/.m2/repository/javax/inject/javax.inject/1/javax.inject-1.jar">
```
Comment (Emily-Jiang) [inline dev/com.ibm.ws.cdi.2.0_fat/.classpath]: The dependency is wrong.

File: dev/com.ibm.ws.cdi.2.0_fat/.classpath [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		<attributes>
+			<attribute name="bsn" value="javax.inject:javax.inject"/>
+			<attribute name="type" value="REPO"/>
+			<attribute name="project" value="com.ibm.websphere.javaee.cdi.1.2"/>
+			<attribute name="version" value="1"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="lib" path="C:/Users/IBM_ADMIN/.m2/repository/javax/enterprise/cdi-api/1.2/cdi-api-1.2.jar">
```
Comment (Emily-Jiang) [inline dev/com.ibm.ws.cdi.2.0_fat/.classpath]: wrong path

File: dev/com.ibm.ws.cdi.2.0_fat/test-applications/beanManagerLookupApp/src/helloworldApp/web/HelloBean.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import javax.enterprise.context.RequestScoped;
+import javax.enterprise.inject.spi.Bean;
+import javax.enterprise.inject.spi.BeanManager;
+import javax.naming.InitialContext;
+
+@RequestScoped
+public class HelloBean {
+
+    public String hello() {
+        return "Hello World CDI 1.2!";
```
Comment (Emily-Jiang) [inline dev/com.ibm.ws.cdi.2.0_fat/test-applications/beanManagerLookupApp/src/helloworldApp/web/HelloBean.java]: change this to CDI 2.0

**Comment by neilgsyoung** [comment]
#build

**Comment by neilgsyoung** [comment]
#build

**Pull request merged**